packages/devtools/src/graphQlDirectives/commitid.scalar.ts

```diff
@@ -0,0 +1,6 @@
+import { GraphQLScalarType } from 'graphql'
+
+export const CeramicCommitID = new GraphQLScalarType({
```

Comment:
Would it make sense to use `CeramicCommitID` from `@glazed/graph/scalars.js` at this point?

Reply:
Yes I'm working on another branch to share scalars between the runtime and devtools packages, I'll open a PR after this on is merged.